ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -28,6 +30,51 @@ def _create_event(**kwargs):
 class ClickhouseTestInsights(
     ClickhouseTestMixin, LicensedTestMixin, insight_test_factory(_create_event, _create_person)  # type: ignore
 ):
+    def test_insight_trends_basic(self):
+        with freeze_time("2012-01-14T03:21:34.000Z"):
+            _create_person(distinct_ids=["1"], team=self.team)
+            _create_person(distinct_ids=["2"], team=self.team)
+            _create_event(team=self.team, event="$pageview", distinct_id="1")
+            _create_event(team=self.team, event="$pageview", distinct_id="2")
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+            response = self.client.get(
+                f"/api/projects/{self.team.id}/insights/trend/?events={json.dumps([{'id': '$pageview'}])}"
+            ).json()
+
+        self.assertEqual(response["result"][0]["count"], 2)
+        self.assertEqual(response["result"][0]["action"]["name"], "$pageview")
+
+        self.assertEqual(response["result"][0]["data"][-2], 2)
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+            response = self.client.get("/" + response["result"][0]["persons_urls"][-2]["url"]).json()
+
+        self.assertEqual(len(response["results"][0]["people"]), 2)
+
+    @test_with_materialized_columns(["key"])
+    def test_breakdown_with_filter(self):
```

Comment:
This materialized test will fail because the person query seems to be using an unmaterialized prop column

Reply:
What's the query? Worth adding the new @snapshot_ decorator that would make this clear?